### ISSUE TITLE
"Viewing only" notes can be edited on web but don't submit edits

### ISSUE THREAD

**Issue by CR0CKER** [standardnotes/collab-editor#4]
View-only URLs create the impression that the document can be edited, but edits are not feeding back into the document. That's confusing. Better don't allow users to edit text and don't give them a curser that allows the viewer to type.

**Comment by atmoio** [comment]
True, I left that in there for the beginning so that people can try it out. I'll take out fake-editing soon.
